Lib/test/test_os/test_os.py

```diff
@@ -1494,6 +1494,76 @@ def test_reload_environ(self):
             self.assertNotIn(b'test_env', os.environb)
             self.assertNotIn('test_env', os.environ)
 
+    def test_clear_empties_environ(self):
+        os.environ["test_key_to_clear1"] = "test_value_to_clear1"
+        os.environ["test_key_to_clear2"] = "test_value_to_clear2"
+        os.environ["test_key_to_clear3"] = "test_value_to_clear3"
```

Comment:
I don't think that 3 variables are useful. IMO a single variable to make sure that os.environ is non-empty is enough.

```suggestion
        os.environ["test_key_to_clear"] = "test_value_to_clear"
```

Reply:
This is to also test the edge case when an errorneous `os.environ.clear()` removes only 1 variable like `os.environ.popitem()` would do.